src/Components/Organization/Card.js

```diff
@@ -0,0 +1,16 @@
+import { styled } from '@mui/material/styles';
+import React from 'react';
+
+export const Card = ({ children }) => {
+  return <Root>{children}</Root>;
+};
+
+const Root = styled('div')(({ theme }) => ({
+  width: '90%',
+  maxWidth: '800px',
+  margin: '30px auto',
+  ...theme.typography.body2,
+  '& > :not(style) + :not(style)': {
+    marginTop: theme.spacing(2),
+  },
+}));
```

Comment:
you can use a material ui card if you want to cretae a Card. If you do it like this Root is too generic, everything can be a root.
- Move Root to Components/CustomBasicComponents/WhateverThisComponentsIsFor
- Rename the Root component
- Change the name from Card to CustomCard to avoid import error with material.

each file should contain only one component

Reply:
Also, this card component is a generic component so it should be on Components/CustomBasicComponents/CustomCard directory

Reply:
I removed the CardCustom Component and used the Card component in MUI